packages/SidePanel/src/components/EscListener.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
```

Comment:
`fyi` I think this change might conflict with @DanilAgafonov takeover migration PR, but might not be that bad.